### DISCUSSION TITLE
Cannot forge pull

### DISCUSSION THREAD

**Discussion by h3dkandi** [magit/forge#441] [Support] [answered]
The error that I get when I try to make a `M-x forge-pull` is:

> ghub-graphql: Symbol’s function definition is void: json-encode

As a matter of fact other commands give the same error as well.
What am I doing wrong?

**Comment by tarsius** [comment, marked as answer]
Just pushed a fix to `ghub`. It can take a few hours until that hits Melpa.

Also see https://github.com/magit/ghub/pull/149.
